code/game/gamemodes/miniantags/abduction/abduction_gear.dm

```diff
@@ -479,6 +479,25 @@ Congratulations! You are now trained for invasive xenobiology research!"}
 		if(BATON_PROBE)
 			to_chat(user, "<span class='warning'>The baton is in probing mode.</span>")
 
+/obj/item/device/radio/headset/abductor
+	name = "alien headset"
+	desc = "An advanced alien headset designed to monitor communications of human space stations. Why does it have a microphone? No one knows."
```

Comment:
First, because I want human looters to be able to use it. Second, because on-mob sprite is waaay cooler when a mic is present.